src/tests/suite/daos_container.c

```diff
@@ -2060,6 +2060,9 @@ co_open_fail_destroy(void **state)
 			    NULL);
 	assert_rc_equal(rc, -DER_IO);
 
+	rc = daos_debug_set_params(arg->group, -1, DMG_KEY_FAIL_LOC,
```

Comment:
If 2061 fails, will we not also cause failures in subsequent tests?

Reply:
ah, ok